cmd/cmd.go

```diff
@@ -36,7 +36,7 @@ var (
 // 2. Create database (if schemaOnly is set to false)
 // 3. Run data conversion (if schemaOnly is set to false)
 // 4. Generate report
-func CommandLine(driver, projectID, instanceID, dbName string, dataOnly, schemaOnly, skipForeignKeys bool, schemaSampleSize int64, sessionJSON string, ioHelper *conversion.IOStreams, outputFilePrefix string, now time.Time) error {
+func CommandLine(driver, projectID, instanceID, dbName string, dataOnly, schemaOnly bool, skipForeignKeys bool, schemaSampleSize int64, sessionJSON string, ioHelper *conversion.IOStreams, outputFilePrefix string, now time.Time, targetDb string) error {
```

Comment:
Nit: Why did we add bool type after schemaOnly?

Reply:
Looks like this got added as a part of some auto correction. I don't recall making this change. Will revert it.